### PR TITLE
fix: TypeError: 'NoneType' object is not subscriptable #416

### PR DESCRIPTION
**Pull request by pritamghanghas**
* we don't want peer book data after the light hotspot switch.

**[Issue](https://github.com/NebraLtd/hm-diag/issues/416)**

- Link:https://github.com/NebraLtd/hm-diag/issues/416
- Summary: remove unncessay call to fetch_miner_data. Leaving the capability behind. Will be removed when miner client switches to gatewayrs

**How**

**Screenshots**

**References**

**Checklist**

- [ ] Tests added
- [ ] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

File: hw_diag/tasks.py [hunk 1]
```diff
@@ -40,14 +38,6 @@ def perform_hw_diagnostics(ship=False):  # noqa: C901
     diagnostics['PK'] = public_keys['key']
     diagnostics['AN'] = public_keys['name']
 
-    # Fetch data from miner container.
-    try:
-        diagnostics = fetch_miner_data(diagnostics)
```
Comment (marvinmarnold) [inline hw_diag/tasks.py:45]: This was the only invocation of `fetch_miner_data`. Can you remove that method all together and related test code?

**Review by marvinmarnold — CHANGES_REQUESTED** [review]

**Review by MuratUrsavas — APPROVED** [review]

File: hw_diag/tasks.py (hunk 1 above)
Comment (pritamghanghas) [reply hw_diag/tasks.py:45]: It was going to be removed with light miner branch. Not sure if helium has any plans to revert on anything. But I can remove it now as well.

**Review by marvinmarnold — APPROVED** [review]

File: hw_diag/tasks.py (hunk 1 above)
Comment (KevinWassermann94) [reply hw_diag/tasks.py:45]: There is no need for backwards compatibility at this point

Comment (pritamghanghas) [reply hw_diag/tasks.py:45]: done

**Pull request merged**